germlineVC.nf

```diff
@@ -679,7 +659,6 @@ def defineToolList() {
     'freebayes',
```

Comment:
ditto freebayes though that can be used for germline, so, if somebody wants to have that, s/he can still use it.

Reply:
Can we use Freebayes for germline?
We're not doing that yet.
Which settings should we use?